pkg/controller/clusterpool/clusterpool_controller.go

```diff
@@ -321,12 +308,10 @@ func (r *ReconcileClusterPool) Reconcile(ctx context.Context, request reconcile.
 
 	availableCapacity := math.MaxInt32
 	if clp.Spec.MaxSize != nil {
-		availableCapacity = int(*clp.Spec.MaxSize) - len(unClaimedCDs) - len(claimedCDs)
+		availableCapacity = int(*clp.Spec.MaxSize) - cds.Total()
 		if availableCapacity <= 0 {
 			logger.WithFields(log.Fields{
-				"UnclaimedSize": len(unClaimedCDs),
-				"ClaimedSize":   len(claimedCDs),
```

Comment:
You're saying someone might grep for "Cannot add more clusters because no capacity is available" and therefore miss the log line above that reports this same information?

Okay, done.